ddtrace/contrib/jinja2/patch.py

```diff
@@ -0,0 +1,131 @@
+import jinja2
+from wrapt import wrap_function_wrapper as _w
+
+from ddtrace import config
+
+from ...ext import http
+from ...utils.formats import asbool, get_env
+from ...pin import Pin
+from ...utils.wrappers import unwrap as _u
+from .constants import DEFAULT_SERVICE, DEFAULT_TEMPLATE_NAME
+
+
+# default settings
+config._add('jinja2',{
+    'service_name': get_env('jinja2', 'service_name', DEFAULT_SERVICE),
+    'inherit_service': asbool(get_env('jinja2', 'inherit_service', True)),
+})
+
+
+def patch():
+    if getattr(jinja2, "__datadog_patch", False):
+        # already patched
+        return
+    setattr(jinja2, "__datadog_patch", True)
+    Pin(
+        service=config.jinja2['service_name'],
+        app='jinja2',
+        app_type=http.TEMPLATE,
```

Comment:
I forgot this. Which `app_type` should be used ?

Reply:
I think we want to just leave it as `None`.

@Kyle-Verhoog ?

Reply:
`app` and `app_type` are deprecated (soon-to-be officially). We can leave it as `None`.